frontend/src/app/data-entry/agent-form/agent-form.component.html

```diff
@@ -19,7 +19,7 @@ <h1 class="mb-4">
             </ng-template>
         </p>
 
-        <lc-form-status [status$]="status$" />
```

Comment:
It turns out the `FormStatus` component can simply retrieve `status$` from the `FormService` by itself!